### DISCUSSION TITLE
It is possible not to use Dynamic Elements in Inline Mode?

### DISCUSSION THREAD

**Discussion by mumus** [sachinchoolur/lightGallery#1188] [Q&A] [answered]
It is possible not to use Dynamic Elements in Inline Mode, i need to make Inline Lightgallery from elements on html markup, for example:
`<div class="inline-gallery-container">`
`    <a href="image1.jpg" data-thumb="thumb1.jpg">`
`        <img src="preview1.jpg" />`
`    </a>`
`<a href="image2.jpg" data-thumb="thumb2.jpg">`
`    <img src="preview2.jpg" />`
`        </a>`
`</div>`

**Comment by sachinchoolur** [comment, marked as answer]
Hey @mumus,

Sorry for the late response.
Here is the demo of inline mode without without using dynamic option  - https://codepen.io/sachinchoolur/pen/qBmKgew

**Reply by mumus** [reply]
Thank u, but how to show thumbs?

**Reply by sachinchoolur** [reply]
Just include thumbnails plugin 

```js
import lgThumbnail from "https://cdn.skypack.dev/lightgallery@2.0.0-beta.4/plugins/thumbnail";
```
```
// include CSS 
// https://cdn.jsdelivr.net/npm/lightgallery@2.0.0-beta.4/css/lg-thumbnail.css
```

```js
plugins: [lgZoom, lgThumbnail]
```

updated demo - https://codepen.io/sachinchoolur/pen/qBmKgew

**Reply by mumus** [reply]
Thank, but why Thumbnails src not working from data-thumb attribute?

**Reply by sachinchoolur** [reply]
Oh. There is an issue with the documentation. If you want to pass thumbnail image via data attribute, you need to use the exThumbImage option 
Please take a look at the docs and let me know if you face any issues  https://www.lightgalleryjs.com/docs/settings/#exThumbImage

**Reply by mumus** [reply]
Thank you very much. It's work fine!!!

**Comment by mumus** [comment]
Hello again. One more question, please.
It is possible to make three types of photos with different sizes, i mean.
1. separate image for thumbnails, for example 60x60 pixels ( exThumbImage, here is ok)
2. separate image for preview in inline not expanded mode, for example, 640x480
3. separate image for expanded gallery, for example 1600 and larger?

**Reply by sachinchoolur** [reply]
Nope. It is currently not possible to use separate images for inline mode and expanded mode. 
lightGallery supports responsive images. Maybe you can try tweaking the srcset and sizes to get the desired behavior 
https://www.lightgalleryjs.com/demos/responsive/
https://stackoverflow.com/questions/61296655/how-to-serve-image-size-based-on-container-not-screen-size

**Reply by mumus** [reply]
Thank u. And one more question. It is possible to change number of thumb in different breakpoints?
For example, i want only 3 thumbs in a row in mobile devices. 
![image](https://user-images.githubusercontent.com/3994560/141452942-9e84ceeb-560d-44cc-8a24-f816610fedf8.png)

**Reply by sachinchoolur** [reply]
You can use `mobileSettings` 
docs - https://www.lightgalleryjs.com/docs/settings/
Change the thumbWidth accordingly - https://www.lightgalleryjs.com/docs/settings/#thumbWidth
